.docs/user-guide/automation.md

```diff
@@ -14,7 +14,32 @@ orchestration tools below.  Optionally, Docker is also listed in some examples.
 
 <br>
 ## Ansible
-ToDo
+With Ansible, installing the latest REX-Ray binaries can be accomplished by
+including the `codenrhoden.rexray` role from Ansible Galaxy.  The role accepts
+all the necessary variables to properly fill out your `config.yml` file.
```

Comment:
@akutz, not sure I follow you here.  This snippet is just an example of what would go in an Ansible playbook.  The `config.yml` file I mention is the REX-Ray YAML file, but the code is what goes to Ansible.  Ansible _generates_ the REX-Ray YAML file from what you give it. Does that help?

As a full example, here is a complete playbook (playbook is just a top-level YAML file you feed to ansible) that I used during testing:

```
travis$ cat rexrayrole.yml
---

- hosts: all
  sudo: yes
  roles:
  - { role: ansible-role-rexray,
      rexray_service: false,
      rexray_storage_drivers: [virtualbox],
      rexray_vbox_endpoint: "http://10.0.2.2:18083",
      rexray_vbox_volume_path: "/Users/travis/VirtualBox VMs/Volumes"
```

You'll see it is at the root -- and is not actually related to the `config.yml` file.